packages/react/src/primitives/shared/constants.ts

```diff
@@ -539,8 +539,9 @@ export const ComponentText = {
     searchButtonLabel: 'Search',
   },
   PasswordField: {
-    showPasswordButtonLabel: 'Show password',
-    hidePasswordButtonLabel: 'Hide password',
+    passwordHidden: 'Password is hidden',
+    passwordShown: 'Password is shown',
+    showPasswordButton: 'Show password',
```

Comment:
nit: I know we've been using "...Label", "...Button" to name text strings, in this file, but when I review the code, I feel it's a little confusing and I'm not able to immidiately know it's the text. Should we just use the verbiage itself, like "showPassword", ""passwordIsHidden", ""passwordIsShown", etc?

Reply:
Yes, I agree that just using the verbiage itself would add some clarify